sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -509,7 +510,12 @@ def test_precompute_invalid_argument():
     X, y, _, _ = build_dataset()
     for clf in [ElasticNetCV(precompute="invalid"),
                 LassoCV(precompute="invalid")]:
-        assert_raises(ValueError, clf.fit, X, y)
+        assert_raises_regex(ValueError, ".*should be.*True.*False.*auto.*"
+                            "array-like.*Got 'invalid'", clf.fit, X, y)
+
+    # Precompute = 'auto' is not supported for ElasticNet
+    assert_raises_regex(ValueError, ".*should be.*True.*False.*array-like.*"
+                        "Got 'auto'", ElasticNet(precompute='auto').fit, X, y)
```

Comment:
The error messages differ a bit. The previous case it should say that `'auto'` is one of the valid options, in the current case it is not accepted anymore...